# 发送前转为弧度
# 颈旋转：80-（-80）度 左-右
# 颈前摆：0-90度 上-下
float64 neck_rotation_theta  # 颈旋转关节角度
float64 neck_front_swing_theta # 颈前摆关节角度